src/Options/index.js

```diff
@@ -81,6 +81,9 @@ export interface ParseServerOptions {
   verbose: ?boolean;
   /* Sets the level for logs */
   logLevel: ?string;
+  /* Set the level used internally by Parse Server features
+  :ENV: PARSE_SERVER_LOG_LEVEL_USES */
+  logLevelUses: ?LogLevelUses;
```

Comment:
```suggestion
  /* (Optional) Overrides the log levels used internally by Parse Server to log events. */
  logLevels: ?LogLevels;
```

I'd simplify the name and remove the "uses". The `:ENV:`name  should be generated automatically when you run `npm run definitions`.